lib/apiservers/engine/backends/volume.go

```diff
@@ -198,7 +202,7 @@ func (v *Volume) VolumeRm(name string) error {
 			return derr.NewRequestNotFoundError(fmt.Errorf("Get %s: no such volume", name))
 
 		case *storage.RemoveVolumeConflict:
-			return derr.NewRequestConflictError(fmt.Errorf("Volume '%s' is in use", name))
+			return derr.NewRequestConflictError(fmt.Errorf(err.Payload.Message))
```

Comment:
sounds good to me as long as it does not conflict with normal docker behavior. The volume stuff is almost free form though.